Wikipedia/Code/WMFWelcomeAnalyticsViewController.m

```diff
@@ -33,20 +34,42 @@ - (void)viewDidLoad {
         self.toggle.on                                                           = NO;
         [[BITHockeyManager sharedHockeyManager] crashManager].crashManagerStatus = BITCrashManagerStatusAlwaysAsk;
     }
+
+    [self.buttonCaretLeft setTintColor:[UIColor wmf_blueTintColor]];
+    [self.buttonCaretLeft wmf_setButtonType:WMFButtonTypeCaretLeft];
 }
 
-- (IBAction)toggleAnalytics:(id)sender {
-    if ([(UISwitch*)sender isOn]) {
+- (IBAction)toggleAnalytics:(UISwitch*)sender {
+    if ([sender isOn]) {
         [[BITHockeyManager sharedHockeyManager] crashManager].crashManagerStatus = BITCrashManagerStatusAutoSend;
         [[NSUserDefaults standardUserDefaults] wmf_setSendUsageReports:YES];
+        
+        dispatchOnMainQueueAfterDelayInSeconds(1.4, ^{
```

Comment:
@nirzar @montehurd i'm not so sure we should be auto-dismissing from a switch interaction. can you think of any examples where Apple does this? HUI guidelines generally discourage doing things on the user's behalf like this.